src/hooks/useLiveSpecs.ts

```diff
@@ -31,19 +36,17 @@ function useLiveSpecs(specType: string) {
     };
 }
 
-export interface LiveSpecsQuery_spec {
+export interface LiveSpecsQuery_spec extends LiveSpecsQuery {
     id: string;
-    catalog_name: string;
     spec: {
         schema: {
             properties: Record<string, any>;
             required?: string[];
         };
         key: string[];
     };
-    spec_type: string;
 }
-const specQuery = ['id', 'catalog_name', 'spec', 'spec_type'];
+const specQuery = queryColumns.concat(['id', 'spec']);
```

Comment:
Changed this so that the fields that are shared in the type are shared via the query array.